commands/fetch.ts

```diff
@@ -1,27 +1,47 @@
-// @ts-nocheck
-const { downloadFileOrFolder } = require('@hubspot/local-dev-lib/fileMapper');
-const { logger } = require('@hubspot/local-dev-lib/logger');
-const {
-  addConfigOptions,
-  addAccountOptions,
-  addOverwriteOptions,
+import { Argv, ArgumentsCamelCase } from 'yargs';
+import { downloadFileOrFolder } from '@hubspot/local-dev-lib/fileMapper';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { CmsPublishMode } from '@hubspot/local-dev-lib/types/Files';
+import {
   addCmsPublishModeOptions,
-  addUseEnvironmentOptions,
+  addOverwriteOptions,
   getCmsPublishMode,
-  addGlobalOptions,
-} = require('../lib/commonOpts');
-const { resolveLocalPath } = require('../lib/filesystem');
-const { validateCmsPublishMode } = require('../lib/validation');
-const { trackCommandUsage } = require('../lib/usageTracking');
-const { i18n } = require('../lib/lang');
+} from '../lib/commonOpts';
+import { resolveLocalPath } from '../lib/filesystem';
+import { validateCmsPublishMode } from '../lib/validation';
+import { trackCommandUsage } from '../lib/usageTracking';
+import { i18n } from '../lib/lang';
+import { makeYargsBuilder } from '../lib/yargsUtils';
+import {
+  AccountArgs,
+  CommonArgs,
+  ConfigArgs,
+  EnvironmentArgs,
+  YargsCommandModule,
+} from '../types/Yargs';
+
+import { EXIT_CODES } from '../lib/enums/exitCodes';
+import { logError } from '../lib/errorHandlers/index';
 
-const { EXIT_CODES } = require('../lib/enums/exitCodes');
-const { logError } = require('../lib/errorHandlers/index');
+type FetchCommandArgs = {
+  src: string;
+  dest?: string;
+  derivedAccountId?: number;
```

Comment:
Nit, but don't need `derivedAccountId` in here anymore since it's defined in CommonArgs